DQMServices/StreamerIO/plugins/DQMMonitoringService.cc

```diff
@@ -115,8 +112,8 @@ namespace dqmservices {
 
       write_json(mstream_, doc, false);
       mstream_.flush();
-    } catch (...) {
-      // pass
+    } catch (std::exception const& exc) {
+      LogDebug("DQMMonitoringService") << "Exception thrown in outputUpdate method: " << exc.what();
```

Comment:
One more static-analyzer warning (`catch(...)`) is coming from here, but I'm not sure how to fix this one. The change in 3de60256c5f is tentative, and I welcome feedback.